src/MapMakerServer.cpp

```diff
@@ -107,6 +107,7 @@ void MapMakerServer::run()
   while (!shouldStop() &&
          ros::ok())  // ShouldStop is a CVD::Thread func which return true if the thread is told to exit.
   {
+    ROS_INFO("running server");
```

Comment:
Make this more clear. it will just show up in the stream of info messages.  What process is saying this? MCPTAM Map Maker Server now running. Or something like that.